### PR TITLE
feat: defaultSuggestion on search

### LINKED ISSUE #8763: defaultSuggestion on search
continue #8761 

- fix `showDefaultSuggestions` for ahp/ahk
- show popular collection (same as sidebar filter in explore items)

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #8763

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

https://github.com/kodadot/nft-gallery/assets/44554284/7180917a-08b9-47e7-9ce3-51d85f0a79db

## Copilot Summary
copilot:summary

copilot:poem

### CODE REVIEW

**Comment by prury** [comment]
two things: i think now we are using Ahp instead of AHP on chain prefixes

won't work if you go inside a collection: 

https://github.com/kodadot/nft-gallery/assets/36627808/88341fb1-6caa-4cbe-932b-78210eadcb85

File: components/search/SearchBar.vue [hunk 1]
```diff
@@ -106,7 +107,7 @@ const placeholderContent = computed(() =>
 )
 
 const showDefaultSuggestions = computed(
-  () => urlPrefix.value === 'rmrk' || urlPrefix.value === 'bsx',
+  () => isRemark.value || isBasilisk.value || isAssetHub.value,
```
Comment (roiLeo) [inline components/search/SearchBar.vue:110]: remove bsx

**Review by roiLeo — APPROVED** [review]
let's reuse same gql query as `TopCollection`

otherwise lgtm

**Review by roiLeo — COMMENTED** [review]
Smoll bug


https://github.com/kodadot/nft-gallery/assets/9987732/85a051b8-7b24-4307-895b-ed6af9742374

see collection description not updating

**Comment by prury** [comment]
2 more things:
- Better text truncation for big collection names:  
[1](https://github.com/kodadot/nft-gallery/assets/36627808/248f89de-c43b-413b-85e5-e065173df53c), [2](https://github.com/kodadot/nft-gallery/assets/36627808/1c0ae8b2-8809-4880-b7e5-80a100986eb3)


- Since search suggestion only works on ahp, ahk and kusama, and only kusama has /series-insight enabled, and /series-insight its the same as top collections, i would remove the rankings link 
image

**Comment by yangwao** [comment]
I would merge it for todays release and can do follow up

**Comment by yangwao** [comment]
cc @hassnian let's resolve conflicts and it will go for release 👌😌

**Comment by yangwao** [comment]
> 2 more things:

We can do it in follow up

**Comment by prury** [comment]
> > 2 more things:
> 
> We can do it in follow up

okok

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 7.7 USD/DOT ~ 6.494 $DOT
🧗 [13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh](https://polkadot.subscan.io/account/13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)
🔗 [0x7a49dd3aa3ac4ab96aed14d65a606bb5806e4732950091f275b5ef3d534cff3e](https://polkadot.subscan.io/extrinsic/0x7a49dd3aa3ac4ab96aed14d65a606bb5806e4732950091f275b5ef3d534cff3e)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**